src/host/PtySignalInputThread.hpp

```diff
@@ -60,7 +60,7 @@ namespace Microsoft::Console
             uint64_t handle;
         };
 
-        [[nodiscard]] HRESULT _InputThread();
+        [[nodiscard]] HRESULT _InputThread() noexcept;
         bool _GetData(_Out_writes_bytes_(cbBuffer) void* const pBuffer, const DWORD cbBuffer);

```

Comment:
```suggestion
        [[nodiscard]] bool _GetData(_Out_writes_bytes_(cbBuffer) void* const pBuffer, const DWORD cbBuffer);
```
Looks like everybody explicitly handles `_GetData()`'s return val (which imo is much better). You could add the `nodiscard` attribute if you want.